### ISSUE TITLE
question on tooling for translators

### ISSUE THREAD

**Issue by Hotell** [facebookincubator/fbt#60]
Hey folks!

I got 2 "breaking" questions to consider this in our next project:

- Does this somehow support exporting to `.mo` files and vice versa ? 
- If not is there any open source tooling that you're using for translators ?

Cheers

**Comment by w1106c** [comment]
Thanks for reaching out to us Hotell!

1. Currently, we do not support exporting to .mo files but we will look into it and evaluate this option.
2. We do not directly support any open source tooling at this point either. However, the output and input formats are not super complex, so I would imagine it to not be too hard to make these work with open source tools.

**Comment by AndrewCraswell** [comment]
@w1106c, @jrwats I'm getting ready to localize an application that we would like to use FBT for and this question is super relevant. I'm interested in your experience working with vendors to do the actual translations. Were FBT's JSON files designed to be compatible with some of the more popular translation file formats for tools used by translators, or is it a brand new standard? Or do you just give the raw JSON to translators and have them edit the same files inline?

**Comment by jrwats** [comment]
@AndrewCraswell, we actually have an internal tool to which vendors have access and through which they submit translations.  They're shown the source string and description (as seen in the JSON output), and for any given parameter, they have the option to specify different strings depending on the variation of the parameter.  These variations are determined on a per-locale basis.

We don't show them raw JSON

**Comment by jrwats** [comment]
I'll take today to look at how difficult it'd be to produce a POT file from FBT collection and similarly, ingest a PO file(s) for translation.

**Comment by AndrewCraswell** [comment]
That would be incredible. I think if something along those lines could become a feature than this library would be a perfect replacement for many existing localization libraries.

**Comment by jrwats** [comment]
Yeah it //appears// pretty straightforward from what I read so far.  I'm
hoping to get away with making POT and PO files in python brace format.
